AnimationSystem/AnimationSystem.cc

```diff
@@ -987,6 +995,33 @@ namespace AnimationSystem {
     }
   }
 
+  void _blendReadyGrab(AnimationMapping &spec, Avatar *avatar) {
```

Comment:
Don't add behavior logic in the C++. This looks like it belongs in JS.

Reply:
Do you mean the whole `_blendReadyGrab()` function ?
I think these `_blendXXX` functions are the most performance comsuming codes.
They includes `animation evaluations`, `lerp/slerp interpolations`, `quaternion multiplications` etc.
If don't have these logics in C++, then what's the meaning of porting Animation System to C++?